grakn-test/src/test/java/ai/grakn/test/engine/cache/EngineCacheTest.java

```diff
@@ -42,25 +45,33 @@
 
 //NOTE: This test is only in grakn-test because it needs a running ZK
 //Ideally this should be moved to the grakn-engine module
-public class EngineCacheDistributedTest {
-    private ConceptCache cache;
+@RunWith(Theories.class)
+public class EngineCacheTest {
 
     //We do this just so we can start ZK
     @ClassRule
     public static final EngineContext engine = EngineContext.startSingleQueueServer();
```

Comment:
It only needs ZK not kafka. Basically my point here is that all this engine test setup code should probably be moved to engine (at some point) so we can start breaking down tests more modularly. Right now grakn-test is becoming a monster. I wanted to move this test into grakn-engine but then I realised I can't because all the setup is in `grakn-test`